packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -164,7 +161,6 @@ const requestRemoteNode = (url, headers, tmpFilename, httpOpts) =>
  */
 async function processRemoteNode({
   url,
-  store,
```

Comment:
need to update docs that `store` is no longer required

Reply:
Done